lib/tasks/db.rake

```diff
@@ -20,8 +20,8 @@ namespace :db do
   task import_to_uat: :environment do
     raise(ArgumentError, "Cannot construct DB connection string") if build_postgres_url.length < 25
 
-    drop_existing_schema = "psql #{build_postgres_url} -c 'drop schema public cascade'"
-    create_new_schema = "psql #{build_postgres_url} -c 'create schema public'"
+    drop_existing_schema = "PGOPTIONS='--client-min-messages=warning' psql -q  #{build_postgres_url} -c 'drop schema public cascade'"
+    create_new_schema = "PGOPTIONS='--client-min-messages=warning' psql -q  #{build_postgres_url} -c 'create schema public'"
```

Comment:
query: should it be `client-min-messages` or `client_min_messages`?

Reply:
🤔 Weird.  You are right that the docs say underscore, but the blog post (and stackoverflow example I found) suggest dashes.
As it seems to accept either,  I will update both this and the db_export to use the underscored version from the postgres docs in case it's an old affectation that gets removed :+1: